dotcom-rendering/src/types/content.ts

```diff
@@ -747,6 +747,8 @@ export interface CampaignFieldText extends CampaignField {
 
 export interface CampaignFieldTextArea extends CampaignField {
 	type: 'textarea';
+	minlength?: string;
+	maxlength?: string;
```

Comment:
by adding these fields, we are able to make use of the exisiting CampaignFieldTextArea interface for message us

Reply:
Was just testing this locally and realised that frontend `minlength` & `maxlength` are typed as `Int` so the parsing validation is failing in DCR. So changing thse to be `number` should fix it

Reply:
nice, all updated.